security/src/test/scala/pl/touk/nussknacker/ui/security/oauth2/ExampleOAuth2ServiceFactory.scala

```diff
@@ -3,7 +3,8 @@ package pl.touk.nussknacker.ui.security.oauth2
 import com.typesafe.scalalogging.LazyLogging
 import io.circe.generic.JsonCodec
 import io.circe.generic.extras.{Configuration, ConfiguredJsonCodec, JsonKey}
-import pl.touk.nussknacker.ui.security.api.{AuthenticatedUser, AuthenticationConfiguration}
+import pl.touk.nussknacker.restmodel.AuthenticatedUser
```

Comment:
It looks like the `security` module is also an API-related module, so we need to add a proper entry to the Migration Guide (see docs)